src/slash_commands/mod.rs

```diff
@@ -50,41 +51,49 @@ impl SlashCommands {
         }
     }
 
+    fn get_command(&self, command_name: &str) -> Option<&Box<dyn SlashCommand>> {
+        self.commands.iter().find(|command| command.get_name() == command_name)
+    }
+
+    async fn send_error_message(ctx: &Context, command: &ApplicationCommandInteraction, error_message: &str) {
+        let error_response_result = command
+            .create_interaction_response(&ctx.http, |response| {
+                response
+                    .kind(InteractionResponseType::ChannelMessageWithSource)
+                    .interaction_response_data(|data| data.content(error_message))
+            })
+            .await;
+
+        if let Err(err) = error_response_result {
+            println!("Could not send error response: {:?}", err);
+        }
+    }
+
+    async fn handle_slash_command(
+        slash_command: &Box<dyn SlashCommand>,
+        ctx: &Context,
+        command: &ApplicationCommandInteraction,
+        chombot: &Chombot,
+        requested_command_name: &str,
+    ) -> Result<(), String> {
+        if let Err(e) = slash_command.handle(&ctx, &command, chombot).await {
+            println!("Handler error for command {}: {:?}", requested_command_name, e);
+            Err(String::from(format!("Could not generate response:\n```\n{}\n```", e)))
```

Comment:
Is `String::from` necessary here? `format!` macro returns `String` instance already AFAIR.